sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -135,6 +135,15 @@ export interface CertificateOperationError {
     readonly message?: string;
 }
 
+// @public
+export type CertificateOperationPollerLike = PollerLike<CertificateOperationPollOperationPublicState, KeyVaultCertificateWithPolicy>;
```

Comment:
Operation...Operation?

Reply:
a typo again :p I was trying to be fast 😁

Reply:
CertificateOperationPollOperation is a bad name, I agree. But, this is the PollOperationState of  the CertificateOperationPoller 😅 help is appreciated.

Reply:
Based on our chat, I'll shorten these names!